cmake/kokkos_enable_options.cmake

```diff
@@ -48,6 +48,7 @@ KOKKOS_ENABLE_OPTION(COMPILER_WARNINGS    OFF "Whether to print all compiler war
 KOKKOS_ENABLE_OPTION(PROFILING_LOAD_PRINT OFF "Whether to print information about which profiling tools got loaded")
 KOKKOS_ENABLE_OPTION(TUNING               OFF "Whether to create bindings for tuning tools")
 KOKKOS_ENABLE_OPTION(AGGRESSIVE_VECTORIZATION OFF "Whether to aggressively vectorize loops")
+KOKKOS_ENABLE_OPTION(VIEW_HOOKS           OFF "Whether to enable View Hooks")
```

Comment:
Related to the above. Why is this off by default? Should we have an option at all?

Reply:
same response as above

Reply:
option removed 60b42a7d87599a9d2c2bfe913cb811b38a3b6f9f